src/common/Loading/index.js

```diff
@@ -0,0 +1,9 @@
+import { Wrapper, StyledSpiner } from "./styled";
+
+const Loading = () => {
+    <Wrapper>
+        <StyledSpiner />
+    </Wrapper>
+};
```

Comment:
Jak nie używamy return(...), to żeby komponent pojawił się na stronie, trzeba użyć okrągłego nawiasu ( ), nie klamrowego { }, czyli 
```
const Loading = () => (
    <Wrapper>
        <StyledSpiner />
    </Wrapper>
);
```
(tak samo dla NoResults)